freud/diffraction.py

```diff
@@ -866,18 +882,22 @@ def plot(self, ax=None, cmap="afmhot", vmin=None, vmax=None):
         Returns:
             (:class:`matplotlib.axes.Axes`): Axis with the plot.
         """
-        if not _HAS_MPL:
-            raise ImportError(msg_mpl)
         if vmin is None:
             vmin = 4e-6 * self.N_points
+
         if vmax is None:
             vmax = 0.7 * self.N_points
+
+        import freud.plot
+
         return freud.plot.diffraction_plot(
             self.diffraction, self.k_values, self.N_points, ax, cmap, vmin, vmax
         )
 
     def _repr_png_(self):
         try:
+            import freud.plot
+
```

Comment:
To appease the pre-commit and the linter you want to revert the changes on the conditional imports to the code that is current with the main branch.